test-configs.yaml

```diff
@@ -574,6 +574,11 @@ device_types:
             - 'multi_v5_defconfig'
             - 'allmodconfig'
 
+  imx6q-var-dt6customboard:
```

Comment:
Tested OK on staging: https://staging.kernelci.org/test/plan/id/5e9820096e1153ea83b2b41a/
Device type in upstream LAVA.